.github/workflows/cross.yml

```diff
@@ -151,16 +172,20 @@ jobs:
         id: toolchain
         with:
           toolchain: stable
-          target: x86_64-pc-windows-gnu
-      - name: Debug Test on `x86_64-pc-windows-gnu`
-        run: cargo test -p aws-lc-rs --target x86_64-pc-windows-gnu --features bindgen
-      - name: Release test on `x86_64-pc-windows-gnu`
-        run: cargo test -p aws-lc-rs --release --target x86_64-pc-windows-gnu --features bindgen
+          target: ${{ matrix.target }}
+      - name: Debug Test on `${{ matrix.target }}`
+        shell: bash
+        run: cargo test -p aws-lc-rs --target ${{ matrix.target }} --features bindgen
+      - name: Release test on `${{ matrix.target }}`
+        shell: bash
+        run: cargo test -p aws-lc-rs --release --target ${{ matrix.target }} --features bindgen
 
   aws-lc-rs-windows-arm64:
     if: github.repository_owner == 'aws'
     name: aarch64-pc-windows-msvc
     runs-on: windows-latest
+    env:
+      RUSTFLAGS: '-Clink-arg= /WX'
```

Comment:
So it looks like `RUSTFLAGS: -Clink-arg=/WX` works (without the space), but not on bash.